docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -57,3 +57,21 @@ If you want to self-host the Adyen app, reach out to [our team <Mail size={12}/>
     - `AUTHORIZATION_ACTION_REQUIRED`
     - `CHARGE_FAILURE`
     - `AUTHORIZATION_FAILURE`
+
+## Limitations & troubleshooting
+
+### Broken channel-configuration mapping
+
+App stores it's configuration assigned to specific Saleor channel.
+
+If you change the assignment, all enqueued notifications (pending notifications from Adyen) that were related to such configuration will be not found by the app, causing the request to fail.
+
+Before you change the configuration mapping, ensure that Adyen webhooks are not pending.
```

Comment:
I don't think you can check pending notifications in Adyen dashboard :/ - maybe we can find some better strategy, for example:
- create new channel in Saleor
- create new config in Adyen app, but making sure that you have the same webhook configuration (i.e. HMAC key, username, password)

or... just never do this / be aware that previous webhook might start failing due to this change (stop accepting payments, give it a few days, then migrate?)